### PR TITLE
PS-4673 Documented how to enable and disable slow_query_log

### PR DESCRIPTION
**Pull request by patrickbirch**
modified file diagnostics/slow_extended
added sections to enable and disable the slow_query_log

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for approval or comments

File: doc/source/diagnostics/slow_extended.rst
Comment (oleksandr-kachan) [inline doc/source/diagnostics/slow_extended.rst:30]: Shall we also describe --slow_query_log parameter which may be passed to mysqld during server startup?

Comment (patrickbirch) [reply doc/source/diagnostics/slow_extended.rst:30]: Isn't that described in line 37?

Comment (patrickbirch) [reply doc/source/diagnostics/slow_extended.rst:30]: Added a note pointing to the MySQL documentation.

**Review by dutow — APPROVED** [review]
Approved - but I'm not sure if we still need changes to the 5.6 documentation.

**Pull request closed without merging**
